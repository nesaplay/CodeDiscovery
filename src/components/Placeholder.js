import React from 'react'

const Placeholder = () => {
	return (
		<div>
			<h2>Welcome to Code Discovery!</h2>
			<p>Use the search input above and discover GitHub repositories</p>
		</div>
	)
}

export default Placeholder
