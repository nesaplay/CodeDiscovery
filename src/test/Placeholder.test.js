import React from 'react'
import Placeholder from '../components/Placeholder'

it('renders a component', () => {
	const wrapper = shallow(
        <Placeholder />
    )

	expect(wrapper).toMatchSnapshot()
})
