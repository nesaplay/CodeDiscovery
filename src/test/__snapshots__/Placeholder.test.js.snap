// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a component 1`] = `
<div>
  <h2>
    Welcome to Code Discovery!
  </h2>
  <p>
    Use the search input above and discover GitHub repositories
  </p>
</div>
`;
